pipelines/push-to-external-registry/README.md

```diff
@@ -19,6 +19,9 @@ Tekton pipeline to release Snapshots to an external registry.
 | taskGitUrl               | The url to the git repo where the release-service-catalog tasks to be used are stored                  | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision          | The revision in the taskGitUrl repo to be used                                                         | No       | -             |
 
+## Changes in 5.3.0
+* Add run-file-updates task
```

Comment:
I'm still not sold on why this is being added to this pipeline vs a new pipeline.
1) Now we have no pipeline to just simply push to another registry
2) The addons flow doesn't require pyxis or signing or anything else from the other pipelines?
3) This will completely break e2e. The e2e test for this pipeline doesn't run on the stage cluster, so InternalRequests will not work for it

Reply:
I have no problem moving this to its own pipeline. thanks for the feedback.

Reply:
> The addons flow doesn't require pyxis or signing or anything else from the other pipelines?

correct.